text/rich-schemas/README.md

```diff
@@ -76,6 +76,73 @@ schema property that may be signed so that an issuer will have the
 ability to canonically specify how the data they wish to sign maps to
 the signature they provide.
 
+## Use of JSON-LD
+Rich schema objects primarily wish to benefit from the accessibility of
+ordinary JSON, but require more sophisticated JSON-LD-driven patterns when
+the need arises.
+
+Each rich schema object will specify the extent to which it supports
+JSON-LD functionality, and the extent to which JSON-LD processing may be
+required.
+
+### What the Casual Developer Needs to Know
+
+* __The `@` character in rich schema objects is reserved for JSON-LD-isms__.
+Any usage of JSON keys that begin with this character is required to be
+JSON-LD-compatible, and any time you see it, you are seeing JSON-LD at
+work.
+
+* __`@context` and `@type` are required at the root of every object__. The
+meaning of these fields in rich schema objects matches JSON-LD's
+expectations, but you don't need to learn JSON-LD to use them.
+
+* __JSON-LD's more advanced mechanisms are an option__--not invoked ad hoc
+for every type of rich schema object, but specified in the formal
+description of each rich schema object. Most rich schema objects use no
+more JSON-LD than `@context`, `@type`, and `@id`.
+
+### Details
+
+Compatibility with JSON-LD was evaluated against version 1.1 of the
+JSON-LD spec, current in early 2019. If material changes in the spec
+are forthcoming, a new analysis may be worthwhile. Our current
+understanding follows.
+
+#### `@type`
+
+The type of an rich schema object, or of an embedded object within a rich
+schema object, is given by the JSON-LD `@type` property.
+[JSON-LD requires this value to be an IRI](https://w3c.github.io/json-ld-syntax/#typed-values).
```

Comment:
This reference looks incorrect to me: https://w3c.github.io/json-ld-syntax/#typed-values. Not sure if this one is better for what you intend: https://w3c.github.io/json-ld-syntax/#node-identifiers